### DISCUSSION TITLE
Having issues while processing any libreoffice file.

### DISCUSSION THREAD

**Discussion by kcubeterm** [kcubeterm/achoz#36] [General]
I'm getting exactly this error https://github.com/harshankur/officeParser/issues/2#issuecomment-901106140

Either we need to add these file type in WIP for now or simply look for another library to process those files.

**Comment by cauerego** [comment]
sounds like we'll do both. wip for now sure is fine. 🤣


-------- Original Message --------
From: Krishna Kanhaiya ***@***.***>
Sent: 28 January 2022 10:58:03 WET
To: kcubeterm/achoz ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [kcubeterm/achoz] Having issues while processing any libreoffice file. (Discussion #36)

I'm getting exactly this error https://github.com/harshankur/officeParser/issues/2#issuecomment-901106140


Either we need to add these file type in WIP for now or simply look for another library to process those files.

**Reply by kcubeterm** [reply]
This PR will ignore these files. 
https://github.com/kcubeterm/achoz/pull/37
